fedmsg/core.py

```diff
@@ -379,6 +382,7 @@ def _create_poller(self, topic="", passive=False, **kw):
             # it appears in the endpoints list due to a hack where it gets
             # added in __init__ above.
             if _name == 'relay_inbound':
+                self.log.debug("ignored relay_inbound from config.endpoints")
```

Comment:
Same deal as above. I'd definitely accept a PR that improves the logging, but it's not really related to the other stuff in this PR.